website/screens/principles/themes/ThemesPage.tsx

```diff
@@ -56,999 +60,1106 @@ const sections = [
-                  <br />
-                  <br />
-                  <Code>placeholderFontColor</Code> (50% of opacity)
-                </td>
-              </tr>
-            </tbody>
-          </DxcTable>
-        ),
-      },
-      {
-        title: "Toggle group",
+        title: "Opinionated Theme",
         content: (
-          <DxcTable>
-            <thead>
-              <tr>
-                <th>Theme Input</th>
-                <th>Tokens (calculation)</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td>Selected base color</td>
-                <td>
-                  <Code>selectedBackgroundColor</Code>
-                  <br />
-                  <br />
-                  <Code>unselectedActiveBackgroundColor</Code>
-                  <br />
-                  <br />
-                  <Code>selectedHoverBackgroundColor</Code> (-8% of lightness)
-                  <br />
-                  <br />
-                  <Code>selectedActiveBackgroundColor</Code> (-18% of lightness)
-                  <br />
-                  <br />
-                  <Code>selectedDisabledBackgroundColor</Code> (+57% of
-                  lightness)
-                </td>
-              </tr>
-              <tr>
-                <td>Selected font color</td>
-                <td>
-                  <Code>selectedFontColor</Code>
-                  <br />
-                  <br />
-                  <Code>selectedDisabledFontColor</Code> (+42% of lightness)
-                </td>
-              </tr>
-              <tr>
-                <td>Unselected base color</td>
-                <td>
-                  <Code>unselectedBackgroundColor</Code>
-                  <br />
-                  <br />
-                  <Code>unselectedHoverBackgroundColor</Code> (-8% of lightness)
-                </td>
-              </tr>
-              <tr>
-                <td>Unselected font color</td>
-                <td>
-                  <Code>unselectedFontColor</Code>
-                </td>
-              </tr>
-            </tbody>
-          </DxcTable>
+          <>
+            <DxcParagraph>
+              This theming strategy lets you change some decisions of the design
+              system, but is still opinionated about the ones we consider
+              fundamental. The list of configurable properties is small, but it
+              applies at the component level.
+            </DxcParagraph>
+            <DxcAlert type="info" size="fillParent">
+              We strongly recommend using the{" "}
+              <Link href="/theme-generator/opinionated-theme" passHref>
+                <DxcLink>opinionated theme generator</DxcLink>
+              </Link>{" "}
+              to create the theme, but you can also create it yourself from
+              scratch.{" "}
+            </DxcAlert>
+            <DxcParagraph>
+              Either through the builder or by creating it from scratch, you
```

Comment:
Maybe it's a good idea to change "builder" and instead use "generator" or "theme-generator" to emphasize the new name.

Reply:
Done!